apps/src/templates/sectionsRefresh/QuickAssignTable.jsx

```diff
@@ -1,8 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import moduleStyles from './sections-refresh.module.scss';
 
 export default function QuickAssignTable({marketingAudience, courseOfferings}) {
-  return <div>{JSON.stringify(courseOfferings[marketingAudience])}</div>;
+  const TABLE_COUNT = Object.keys(courseOfferings[marketingAudience]).length;
+
+  const renderTable = headerIndex => {
+    return (
+      <table className={moduleStyles.table}>
+        <thead>
+          <tr className={moduleStyles.headerRow}>
+            <td className={moduleStyles.headerCell}>
+              <div>
+                {Object.keys(courseOfferings[marketingAudience])[headerIndex]}
```

Comment:
This doesn't need to be done in this PR but I think we'll need to translate the curriculum types to the correct table headings. ie we use "Course" as a curriculum type but the header of the column should be "Courses".

It does make me wonder if it's worth hardcoding in the curriculum types more. ie
```
const renderTable = (key, title) => {
 // The code you already have
}

  return (
    <div className={moduleStyles.multiTables}>
       {!!courseOfferings[marketAudience]['Course'] && renderTable('Course', 'Courses')}
       {!!courseOfferings[marketAudience]['Module'] && renderTable('Module', 'Modules')}
       {!!courseOfferings[marketAudience]['Standalone Unit'] && renderTable('Standalone Unit', 'Standalone Units')}
    </div>
);
```

Ideally, you could use the constants we already have. This might impact how the HoC and PL menus are built though, so it might not be a better direction.

Reply:
Ohh this is a good point. Okay will update- also I imagine I should make the titles translatable too?

Reply:
Oh good call! Obviously not in this PR but translatability should be something we should think about before launch.

Reply:
updated the constants to include hoc and pc, and added translations